src/ekklesia_notify/settings_models.py

```diff
@@ -15,16 +15,22 @@ class ClientSettings(BaseSettings):
     allowed_senders: List[str]
     allowed_templates: List[str]
 
+    class Config:
```

Comment:
could we use a base class for our settings models to avoid duplicationg the Config inner class?